eva-tools/src/main/java/embl/ebi/variation/eva/vcfDump/VariantExporter.java

```diff
@@ -351,4 +591,61 @@ public int getFailedVariants() {
         }
         return variantContextMap;
     }
+
+    private String getContextNucleotideFromCellbase(Variant variant, Integer start, String studyId) throws IOException {
+        if (cellbaseClient != null) {
+            String nucleotide = null;
+            List<Region> regions = Collections.singletonList(new Region(variant.getChromosome(), start - 1, start-1));
+
+            QueryResponse<QueryResult<GenomeSequenceFeature>> sequence = cellbaseClient.getSequence(
+                    CellBaseClient.Category.genomic, CellBaseClient.SubCategory.region, regions, null);
+
+            List<GenomeSequenceFeature> response = sequence.getResponse().get(0).getResult();
+            if (response.size() == 1) {
```

Comment:
If this condition is not satisfied, the `nucleotide` variable will be null and lines 545-546 (https://github.com/EBIvariation/eva-v2/pull/13/files#diff-872a43aeafd98e7df90f7808f1c5270cR545) will break. An exception could be thrown and (I think) handled the same way as the IllegalArgumentException below.